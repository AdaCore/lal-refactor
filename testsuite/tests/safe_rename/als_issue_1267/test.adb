procedure Test is
   A, B : Integer;
begin
   declare
      C, D : Integer;
   begin
      declare
         E, F : Integer;
      begin
         B := 1;
   null;
      end;
   end;
   null;
end Test;
