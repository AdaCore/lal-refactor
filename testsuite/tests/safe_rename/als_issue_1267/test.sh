#!/bin/sh
safe_rename -P test.gpr -S test.adb -L 2 -R 7 -N G --algorithm analyse_ast
