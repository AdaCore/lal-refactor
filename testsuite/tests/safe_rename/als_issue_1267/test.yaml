description:
    Refactoring Safe Rename test that confirms that the tool does not crash when
    checking Basic_Decl nodes that contain more than one Defining_Name.

driver: shell_script
