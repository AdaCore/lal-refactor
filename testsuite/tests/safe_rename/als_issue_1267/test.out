# Renaming node test.adb:2:7: 
References:
Unit:test.adb
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 7
End_Line: 2
End_Column: 8
Line: 10
Source Location Range:
Start_Line: 10
Start_Column: 10
End_Line: 10
End_Column: 11

